pkg/koordlet/resmanager/cpu_burst.go

```diff
@@ -325,7 +325,11 @@ func (b *CPUBurst) applyCFSQuotaBurst(burstCfg *slov1alpha1.CPUBurstConfig, podM
 			continue
 		}
 		containerCurCFS, err := b.cgroupReader.ReadCPUQuota(containerPath)
-		if err != nil {
+		if err != nil && system.IsCgroupDirErr(err) {
```

Comment:
system.IsCgroupDirErr çan be ignored inside ReadCPUQuota with V(5) logging, so that the outside part don't need to check everytime.

so does the update part in executor.